README.md

```diff
@@ -100,3 +98,64 @@ pyinfra inventory.py deploy.py
 Now you know the building blocks of pyinfra! By combining inventory, operations and Python code you can deploy anything.
 
 See the more detailed [getting started](https://docs.pyinfra.com/page/getting-started.html) or [using operations](https://docs.pyinfra.com/page/using-operations.html) guides. See how to use [inventory & data](https://docs.pyinfra.com/page/inventory-data.html), [global arguments](https://docs.pyinfra.com/page/arguments.html) and [the CLI](https://docs.pyinfra.com/page/cli.html) or check out the [documented examples](https://docs.pyinfra.com/page/examples.html).
+
+## Using Pyinfra Programtically
```

Comment:
For now I'd like to avoid documenting the API mode, pending decision on supporting it and/or defining how stable it is/isn't.